core/modules/user/user.module

```diff
@@ -936,49 +944,57 @@ function user_account_form(&$form, &$form_state) {
-    '#format' => isset($account->signature_format) ? $account->signature_format : NULL,
+    '#access' => (!$register && ($site_config->get('user_signatures') || $site_config->get('user_pictures'))),
+    '#group' => 'additional_settings',
   );
-
+  // Signature.
+  if ($site_config->get('user_signatures')) {
+    $form['personalization']['signature'] = array(
+      '#type' => 'text_format',
+      '#title' => t('Signature'),
+      '#default_value' => isset($account->signature) ? $account->signature : '',
+      '#description' => t('Your signature will be publicly displayed at the end of your comments.'),
+      '#format' => isset($account->signature_format) ? $account->signature_format : NULL,
+    );
+  }
   // Picture/avatar.
-  $form['picture'] = array(
+  if ($site_config->get('user_pictures')) {
+    $form['personalization']['picture'] = array(
+      '#type' => 'value',
+      '#value' => isset($account->picture) ? $account->picture : NULL,
+    );
+    $form['personalization']['picture_current'] = array(
+      '#markup' => theme('user_picture', array('account' => $account)),
+    );
+    $form['personalization']['picture_delete'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Delete picture'),
+      '#access' => !empty($account->picture->fid),
+      '#description' => t('Check this box to delete your current picture.'),
+    );
+    $form['personalization']['picture_upload'] = array(
+      '#type' => 'file',
+      '#title' => t('Upload picture'),
+      '#size' => 48,
+      '#description' => t('Your virtual face or picture. Pictures larger than @dimensions pixels will be scaled down.', array('@dimensions' => $site_config->get('user_picture_dimensions'))) . ' ' . filter_xss_admin($site_config->get('user_picture_guidelines')),
+    );
+    $form['#validate'][] = 'user_validate_picture';
+  }
+
+  $form['regional_language'] = array(
     '#type' => 'fieldset',
-    '#title' => t('Picture'),
-    '#weight' => 2,
-    '#access' => (!$register && $site_config->get('user_pictures')),
-  );
-  $form['picture']['picture'] = array(
-    '#type' => 'value',
-    '#value' => isset($account->picture) ? $account->picture : NULL,
-  );
-  $form['picture']['picture_current'] = array(
-    '#markup' => theme('user_picture', array('account' => $account)),
-  );
-  $form['picture']['picture_delete'] = array(
-    '#type' => 'checkbox',
-    '#title' => t('Delete picture'),
-    '#access' => !empty($account->picture->fid),
-    '#description' => t('Check this box to delete your current picture.'),
-  );
-  $form['picture']['picture_upload'] = array(
-    '#type' => 'file',
-    '#title' => t('Upload picture'),
-    '#size' => 48,
-    '#description' => t('Your virtual face or picture. Pictures larger than @dimensions pixels will be scaled down.', array('@dimensions' => $site_config->get('user_picture_dimensions'))) . ' ' . filter_xss_admin($site_config->get('user_picture_guidelines')),
+    '#title' => t('Regional and language'),
```

Comment:
Can we change this heading (and the form key `regional_language`) to "Region and Language" (`region_language`)? That would match terminology used by OSes

MacOS:
![image](https://user-images.githubusercontent.com/100206/187354289-a1c6fcf2-4e26-4dbc-aa57-54a6826a0e6e.png)

Windows:
![image](https://user-images.githubusercontent.com/100206/187353766-25b98aee-4ccc-42d0-8979-2e9a4dfb4b84.png)

iPhone:
![image](https://user-images.githubusercontent.com/100206/187353848-0524df03-c398-4611-bdef-32223c025c58.png)

Ubuntu:
![image](https://user-images.githubusercontent.com/100206/187354151-249663a4-2415-4bcb-b202-805879cc48d9.png)


The key `regional_language` led me to believe it was for dialects specific to a particular region (a "regional language").

Reply:
```suggestion
    '#title' => t('Region and language'),
```